src/amo/components/Category/index.js

```diff
@@ -24,7 +18,10 @@ export class CategoryBase extends React.Component {
     category: PropTypes.object,
     dispatch: PropTypes.func.isRequired,
     errorHandler: PropTypes.object.isRequired,
+    filters: PropTypes.object,
     loading: PropTypes.boolean,
+    paginationQueryParams: PropTypes.object,
```

Comment:
Oh, would that count? It's using `ownProps` because the router is passing that to `mapStateToProps`, but the `CategoryBase` component doesn't actually use it at all, just `page`.

Do we include properties used by `mapStateToProps` in the `*Base` component's `PropTypes`? I thought not, because they never actually use `ownProps`/`location`.